java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Clean grub2 reinstall entry in autoyast snippet (bsc#119950)
```

Comment:
This is wrong bug number - https://bugzilla.suse.com/show_bug.cgi?id=1199950
```suggestion
- Clean grub2 reinstall entry in autoyast snippet (bsc#1199950)
```

Reply:
addressed